xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-webjar/src/main/webjar/lightbox.js

```diff
@@ -0,0 +1,293 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+define('lightboxTranslationKeys', {
```

Comment:
```suggestion
define('xwiki-lightbox-messages', {
```

Reply:
@mflorea there's a convention for those names? Couldn't see it in https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HJavaScriptBestPractices. (btw seems we're missing a javascript entry in https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/)

Reply:
@surli it's the second bullet on https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HJavaScriptBestPractices

> XWiki JavaScript modules should have their name prefixed with 'xwiki-' in order to avoid conflicts with external modules

The main point of my suggestion was the ``xwiki-`` prefix. The rest is just my preference.